src/components/NcAppNavigation/NcAppNavigation.vue

```diff
@@ -257,8 +257,7 @@ export default {
 	backdrop-filter: var(--filter-background-blur, none);
 
 	&--close {
-		transform: translateX(-100%);
-		position: absolute;
+		margin-left: -$navigation-width;
```

Comment:
On mobile width can be smaller, for example, ~260px on 320px window size.

So margin here should be set the same as on navigation width - using `max` between 300px and `100vw - (var(--app-navigation-padding) ...` (see line 240).